### PR TITLE
Fix 'test_action_initiate_smartstate_analysis'

### PR DESCRIPTION
**Pull request by vrutkovs**
{{pytest: cfme/tests/control/test_actions.py -k 'test_action_initiate_smartstate_analysis' --use-provider 'vsphere55' -v --long-running -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit daf8773a7b05e346f84cc426bafafe7a2376f85c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dd1ab6e5d61f435a0fe094e8ba2ac805331e2e1a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f09f1fd7fe8128c9407d02e8a23e809771e34c84:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ef94999e8b9ae806e5e0e5225fbc8c5238e17ffd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ede8f3a727d12127d841c46216bfd51e38ef9b75:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/control/test_actions.py [hunk 1]
```diff
@@ -99,14 +99,22 @@ def get_vm_object(vm_name):
     Returns:
         If found, :py:class:`utils.miq_soap.MiqVM`. If not, `None`
     """
-    vm_table = cfmedb()['vms']
```
Comment (psav) [inline cfme/tests/control/test_actions.py]: Documentation above needs updating

File: cfme/tests/control/test_actions.py [hunk 2]
```diff
@@ -163,19 +175,16 @@ def _finalize():
...
-        soap = wait_for(
-            lambda: get_vm_object(vm_name),
-            message="VM object {} appears in CFME".format(vm_name),
-            fail_condition=None,
-            num_sec=600,
-            delay=15,
-        )[0]
-    else:
-        soap = None
+    # Get the SOAP/REST API object
```
Comment (psav) [inline cfme/tests/control/test_actions.py]: Is SOAP still referenced here? perhaps it needs removing from here and in the comment on line 163

**Comment by dajoRH** [comment]
Lint report for commit 84750ac2e9a8aa7c48f6c7a65ac286ad50ed50f9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5a196bbcdce6baeafaefdb05f2fc3e1308558f5d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1d16b2ffa1cee422bd76214753c52fb192a19cfb:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/control/test_actions.py:597:101: E501 line too long (102 > 100 characters)
./cfme/tests/cloud/test_instance_analysis.py:5:1: F401 'error' imported but unused
./cfme/tests/cloud/test_instance_analysis.py:6:1: F401 'TimedOutError' imported but unused
./cfme/tests/cloud/test_instance_analysis.py:86:16: E702 multiple statements on one line (semicolon)
./cfme/tests/cloud/test_instance_analysis.py:88:32: W292 no newline at end of file
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3f9b90bd16e390d1a8ee9b848e30de4d1b666399:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/control/test_actions.py:597:101: E501 line too long (102 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eab836442399f2a44969d0bb5905331f4c4f921a:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/control/test_actions.py:596:55: W291 trailing whitespace
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9aec7dca1c79881ea90adb7e56af120ebe63a4e5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 674b326b3cdfecc2b84048d7c71087ec6a2350ce:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1b0b81b514e6040611951fec4d1e7bdc43b7e777:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 41ccb6f42a64d075e64c89c7ff12ea2899413c53:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a8eba1f7dbd98d8316adca5d3ab4da22c9a97793:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 337b18917aa54c3cc10d161889ce896101a38e3b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit efd24af51b4bcd7def137d6bce2dd10569ea5280:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ecca977b27c6977d4da9b07daf0671eff7bcd230:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d112f6c32f4b67b0ed7b4278eaa6fe752f84137a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b885e681670543fb6a13888b94bc973fab578e8e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b1227f6cf90ce577bacf8dd68b34af5b39bb89cc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cbb73e86e448eccedb01f1aeeabcd8b264f62d8a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 990a4a293391c765952b8edf3b4ce190b7134ffb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b4b48ea5d676d6ed5d42a66c70eac126959f6edb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0b673d445a1edb606f409e1180289ba1cd4d83ce:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/control/test_actions.py:642:101: E501 line too long (104 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 81c90a39617e5170768df0309e503bf9d1e49aac:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 09ce290fbb2495652c12a257c02891bdfb0198d0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 64f7d4db80fbb112336617badb77f9e774710915:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/control/test_actions.py [hunk 3]
```diff
@@ -15,13 +15,19 @@
 # TODO: Move the SOAP calls to UI checks since SOAP is deprecated
 import fauxfactory
 import pytest
+import cfme.fixtures.pytest_selenium as sel
```
Comment (mfalesni) [inline cfme/tests/control/test_actions.py]: Inside the test run, the pytest_selenium is available as `sel` inside the pytest namespace.

so you can write `pytest.sel.click()` and so.

Comment (vrutkovs) [inline cfme/tests/control/test_actions.py]: Right, fixed in the next commit

**Comment by dajoRH** [comment]
Lint report for commit acd721b6da6fddacdc2049636f3cb6dd34d2ebf4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2ba8d77a1f8f16bd355f575b5ea33958c9ce63d6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**